src/main/java/org/launchcode/liftoffrecipetracker/models/Recipe.java

```diff
@@ -3,29 +3,34 @@
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
-import org.launchcode.liftoffrecipetracker.data.RecipeRepository;
-import org.springframework.beans.factory.annotation.Autowired;
 
 import javax.persistence.Entity;
 import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
 
 //@Indexed creates an index of the schema for searching purposes. Only entities with @Indexed will be indexed.
 @Entity
 @Indexed
-public class Recipe extends AbstractRecommendations {
+public class Recipe extends AbstractEntity {
 	//properties
 	@NotBlank
 	@NotNull
 	@Size(min=5, message = "Ingredients are required.")
 	private String ingredients;
 
+	//@FullTextField annotation creates an index of this property that is full-text searchable
+	@NotBlank(message = "Recipe name required.")
+	@NotNull
+	@Size(min=2,max=50, message = "Recipe name must be between 2 and 50 characters long.")
+	@FullTextField
+	private String name;
+
```

Comment:
This is all taken care of in AbstractRecommendations. Shouldn't need to switch the functionality back to this class. Were you having trouble getting this to work with the other abstract class?